release-notes/Cube/Cube_Feature_Release_10.3/Cube_10.3.1.md

```diff
@@ -41,6 +41,16 @@ ChildrenSort="Property|property=Start,asc"
 ChildrenSort="Property|property=End,desc"
 ```
 
+#### Visual Overview - '[GroupValue]' and '[Count]' can now be used as placeholders to adjust the value inside of dynamically grouped shapes [ID_34974]
+
+<!-- MR 10.4.0 - FR 10.3.1 -->
+
+When configuring a Visio drawing, users can now enter the placeholder "[GroupValue]" in the *ShapeGrouping* shape to display the parameter value of the shape data *GroupBy* of the first child of the group. If the *GroupBy* value has multiple column parameters or properties, these will be separated by commas.
```

Comment:
*ShapeGrouping* is actually a shape **data** that resides on a shape.
Reference: https://docs.dataminer.services/user-guide/Basic_Functionality/Visio/positioning_shapes/Enabling_grouping_of_dynamically_positioned_shapes.html?q=ShapeGrouping